engine/src/mac-window.mm

```diff
@@ -2216,7 +2224,15 @@ bool MCMacDoUpdateRegionCallback(void *p_context, const MCRectangle &p_rect)
 	
 	// Force a re-display, this will cause drawRect to be invoked on our view
 	// which in term will result in a redraw window callback being sent.
-	[m_view displayIfNeededInRect: [m_view mapMCRectangleToNSRect:MCRegionGetBoundingBox(m_dirty_region)]];
+	if (![m_delegate inUserReshape])
+	{
+		NSEvent *t_event;
+		t_event = [NSApp nextEventMatchingMask: NSApplicationDefinedMask
+									 untilDate: [NSDate dateWithTimeIntervalSinceNow: 0.1]
```

Comment:
I take it we can't get away with a smaller interval (maybe even zero?) - and if we can, do we still need the sync event?

Reply:
I've tested with lowering the interval - 0.02 ms seems to fix the redraw issues. Lower than that brings them back. That would be consistent with a maximum redraw rate of 60hz

Reply:
@livecodeian The interval here is still 0.1s - but the above statement suggests 0.02 works... Additionally I suggest explaining why 0.02s in the comment above :)